docs/libcurl/opts/CURLOPT_SSH_KEYFUNCTION.3

```diff
@@ -73,6 +74,13 @@ info from libcurl on the matching status and a custom pointer (set with
 \fICURLOPT_SSH_KEYDATA(3)\fP). It MUST return one of the following return
 codes to tell libcurl how to act:
 
+.IP CURLKHSTAT_FINE_REPLACE
+Added in 7.7x.0 The new host+key is accepted and libcurl will replace the old 
```

Comment:
It looks like we can merge this in time for 7.73.0

Reply:
done, soon I will commit